test/narjure/test/concept_creator.clj

```diff
@@ -22,3 +27,30 @@
   (is (= 10 (count @c-map)))
   (is (= true (contains? @c-map (keyword (str 5)))))
   )
+
+; test_1 - initialise test
+; a. initial state is correct
+; b. actor is registered
+
+
+; test_2 - shutdown test
+; a. check :shutdown is handled correctly
+
+
+; test_3 - clean-up test
+; a. ensure all created concepts are shutdown corectly
+
+; test_1 - test processing of :task-msg [task]
+; a. no terms exists as concepts (:terms and :occurrence)
+; b. all terms exisit as concepts (:terms and :occurrence)
+; c. some terms exist as concepts (:terms and :occurrence)
+
+; test_2 - concept-count is maintained correctly
+
+; test_3 concept-limit is reached
+; a. :concept-limit-msg is posted to :forgettable-concept-collator
+
+; test_4 - task is returned to task-dispatcher
+; a. :task-msg [task] is posted to :task-dispatcher
+
```

Comment:
They were causing a problem for circle-ci so I took them out. I am exploring an alternative appraoch